src/main/webapp/WEB-INF/views/car/creation.jsp

```diff
@@ -0,0 +1,21 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: savia
+  Date: 20/01/2021
+  Time: 20:58
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Creation</title>
+</head>
+<body>
+<h1>Please, provide cars details</h1>
+<form method="post" action="${pageContext.request.contextPath}/car/creation">
+    Please, fill car`s model: <input type="text" name="model">
+    and manufacturer id: <input type="text" name="manufacturer_id">
```

Comment:
what if there is no manufacturer with your input id? and what if you input "2hk23ds5" here?

Reply:
Fixed